utils/rpms/packaging/Dockerfile.mockbuild

```diff
@@ -30,7 +30,7 @@ RUN chmod +x /tmp/repo-helper.sh &&                 \
 # This needs to be moved to a shell script like above in the future to
 # properly only remove the proxy variables only when they need to be removed
 RUN dnf -y install mock make                                        \
-                   rpm-build createrepo rpmlint redhat-lsb-core git \
+                   rpm-build createrepo rpmlint git                 \
```

Comment:
in general what is the relationship between the files in the daos repo here, and the files of the same name found in the packaging repo https://github.com/daos-stack/packaging ? Are they intended to be identical under typical circumstances?